internal/storage/storageutil/object_attrs.go

```diff
@@ -147,6 +147,7 @@ func ConvertObjToMinObject(o *gcs.Object) *gcs.MinObject {
 		Updated:         o.Updated,
 		Metadata:        o.Metadata,
 		ContentEncoding: o.ContentEncoding,
+		CRC32C:          o.CRC32C,
```

Comment:
@gargnitingoogle Please let us know if this requires any change in logic to compute size of minObject.

Reply:
Just curious, why is CRC needed to be stored in Stat Cache? Is there a way we can do by not storing in Stat Cache. ( like not storing all properties ).

Reply:
If its not in stat cache, we end up making a stat call every time a file is downloaded.

Reply:
Thanks, Got it. 
To store GCS response crc in memory for async download comparison.

Reply:
Memory computation changes will be done in a seperate PR. Resolving this comment.